mysqld_exporter.go

```diff
@@ -175,14 +186,19 @@ func (e *Exporter) setMetrics(scrapes <-chan []string) {
 			continue
 		}
 
-		if _, ok := e.metrics[name]; !ok {
-			e.metrics[name] = prometheus.NewGauge(prometheus.GaugeOpts{
-				Namespace: namespace,
-				Name:      name,
-			})
-		}
+		command := comRegex.FindStringSubmatch(name)
+		if len(command) > 0 {
+			e.commands.With(prometheus.Labels{"command": command[1]}).Set(float64(value))
```

Comment:
Looking at https://gist.github.com/siavashs/798fa9619b91918d4524 I wonder whether other metrics should also be joined together with labels - for example, `mysql_handler_*`. Any opinion on that?